dll/cpl/access/access.rc

```diff
@@ -34,6 +34,9 @@ IDI_CPLACCESS ICON "resources/applet.ico"
 #ifdef LANGUAGE_EN_US
     #include "lang/en-US.rc"
 #endif
+#ifdef LANGUAGE_ET_EE
```

Comment:
Should be after ES_ES.

Reply:
Right, I usually do that, unsure why I let it slip this time.